crypto/ts/ts_rsp_sign.c

```diff
@@ -631,6 +631,8 @@ static int ossl_ess_add1_signing_cert(PKCS7_SIGNER_INFO *si,
 {
     ASN1_STRING *seq = NULL;
     int len = i2d_ESS_SIGNING_CERT(sc, NULL);
+    if (len <= 0)
```

Comment:
We do not allow intermixed declarations and code.

Reply:
Plus a blank line between declarations and code :(